docs/reference/search/suggesters/phrase-suggest.asciidoc

```diff
@@ -167,33 +167,33 @@ The response contains suggestions scored by the most likely spell correction fir
     to `0.0` the top N candidates are returned. The default is `1.0`.
 
 `max_errors`::
-    the maximum percentage of the terms that at most
+    The maximum percentage of the terms 
     considered to be misspellings in order to form a correction. This method
     accepts a float value in the range `[0..1)` as a fraction of the actual
     query terms or a number `>=1` as an absolute number of query terms. The
-    default is set to `1.0` which corresponds to that only corrections with
-    at most 1 misspelled term are returned.  Note that setting this too high
-    can negatively impact performance. Low values like `1` or `2` are recommended
+    default is set to `1.0`, meaning only corrections with
```

Comment:
I was a bit unsure about the accuracy of this edit.

Reply:
It looks good to me